### ISSUE TITLE
Double Transpose and Cast nodes after converting from tf to onnx inception_v2 model

### ISSUE THREAD

**Issue by LapshinA** [onnx/tensorflow-onnx#815]
**Describe the bug**
Double Transpose and Cast nodes after converting from tf to onnx inception_v2 model

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Tensorflow Version: 1.5
- Python version: 3.6.8

**To Reproduce**
Convert model using:
onnx_graph = tf2onnx.tfonnx.process_tf_graph(graph,
                                             input_names=["import/input:0"],
                                            output_names=["import/InceptionV2/Predictions/Reshape_1:0"],         opset=8)

**Expected behavior**
Double Transpose and Cast nodes after converting from tf to onnx inception_v2 model

**Screenshots**
![fragment](https://user-images.githubusercontent.com/5502685/75566039-1beb7f00-5a60-11ea-8da3-7f236f50f967.png)

**Comment by LapshinA** [comment]
Link to model - http://download.tensorflow.org/models/inception_v2_2016_08_28.tar.gz

**Comment by guschmue** [comment]
which version of tf2onnx was this ? There have been some improvements a few weeks ago. tf2onnx==1.5.5 or master would have those.

**Comment by LapshinA** [comment]
I used 1.55 version
![tf2onnx_version](https://user-images.githubusercontent.com/5502685/75754175-4c236e00-5d3d-11ea-9af3-4f4b96afe6ee.png)

**Comment by jignparm** [comment]
@LapshinA -- can you confirm the link to the model that you posted? It only contains a .ckpt file, and no meta files. How are you able to load the model?

Did you mean this link instead? https://storage.googleapis.com/download.tensorflow.org/models/inception_v2_2016_08_28_frozen.pb.tar.gz

**Comment by LapshinA** [comment]
We use - http://download.tensorflow.org/models/inception_v2_2016_08_28.tar.gz and process it next script:

from tensorflow.python.tools import (freeze_graph as freeze_graph_lib,
                                     optimize_for_inference_lib)

freeze_graph_lib.freeze_graph(
        input_graph=r'D:\Chekpoints\PretrainedModels\inception_v1\inception_v1_inference_graph.pb',
        input_saver='',
        input_checkpoint='D:\Chekpoints\PretrainedModels\inception_v1.ckpt',
        input_binary=True,
        output_graph='D:\Chekpoints\PretrainedModels\inception_v1\inception_v1_optimized_graph.pb',
        output_node_names='InceptionV1/Logits/Predictions/Reshape_1',
        initializer_nodes='',
        filename_tensor_name='save/Const:0',
        restore_op_name='save/restore_all',
        clear_devices=True
        )

**Comment by jignparm** [comment]
@LapshinA , just to be  precise about the model link, where are you getting `inception_v1_inference_graph.pb` from? Based on the log below, the link pasted above does not contain the graphdef file.


### Download log
```
localadmin@JRP-VM1-WIN-GPU MINGW64 /d/temp
$ curl http://download.tensorflow.org/models/inception_v2_2016_08_28.tar.gz > inception_v2_2016_08_28.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 39.6M  100 39.6M    0     0  48.8M      0 --:--:-- --:--:-- --:--:-- 48.8M

localadmin@JRP-VM1-WIN-GPU MINGW64 /d/temp
$ tar xvfz inception_v2_2016_08_28.tar.gz
inception_v2.ckpt

localadmin@JRP-VM1-WIN-GPU MINGW64 /d/temp
$ ll
total 84412
-rw-r--r-- 1 localadmin 197121 44856794 Aug 29  2016 inception_v2.ckpt
-rw-r--r-- 1 localadmin 197121 41577079 Mar 12 18:43 inception_v2_2016_08_28.tar.gz

```

**Comment by jignparm** [comment]
@LapshinA , below are similar nodes to the ones you posted, but using the frozen model from https://storage.googleapis.com/download.tensorflow.org/models/inception_v2_2016_08_28_frozen.pb.tar.gz instead. 

As you can see, there are no double transpose nodes. You need to run the optimizers to eliminate the transpose nodes. However, just to make sure we are on the same page, and not using different models, I will need to know which graphdef file you are using (and not just the .ckpt file).



![image](https://user-images.githubusercontent.com/13698702/76577807-502e5900-64be-11ea-8378-e931538db1d7.png)

**Comment by LapshinA** [comment]
Hello, 
We get pb file according to  https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_Slim_Library_Models.html#example_of_an_inception_v1_model_conversion

Please download pb file which we used: https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_Slim_Library_Models.html#example_of_an_inception_v1_model_conversion

**Comment by jignparm** [comment]
The 2 models uploaded on google drive don't contain  the output = `InceptionV2/Predictions/Reshape_1:0`. Instead they contain  output=`InceptionV1/Logits/SpatialSqueeze:0`.

However, they convert successfully, without any double transpose,  by just using the regular command line of tf2onnx.

###Cmd line
```
python -m tf2onnx.convert --input inception_v1_18022020.sorted.pb --inputs input:0 --outputs InceptionV1/Logits/SpatialSqueeze:0 --opset  11 --output converted.onnx
```

Feel free to close out this issue -- you should be able to run the above to convert your model without double transposes.
